applications/teletype/src/teletype_util.erl

```diff
@@ -70,8 +70,41 @@ send_email(Emails, Subject, RenderedTemplates, Attachments) ->
                | add_attachments(Attachments)
               ]
             },
-    relay_email(To, From, Email).
+    Skip = wh_util:is_true(get('skip_smtp_log')),
+    AccountId = get('account_id'),
+    case relay_email(To, From, Email) of
+        {'ok', Receipt} ->
+            maybe_log_smtp(Skip, AccountId, Emails, Subject, RenderedTemplates, Receipt, 'undefined');
+        {'error', Reason} = E ->
+            maybe_log_smtp(Skip, AccountId, Emails, Subject, RenderedTemplates, 'undefined', wh_util:to_binary(Reason)),
+            E
+    end.
 
+-spec maybe_log_smtp(boolean(), api_binary(), list(), ne_binary(), list(), api_binary(), api_binary()) -> 'ok'.
+maybe_log_smtp('true', _AccountId, _Emails, _Subject, _RenderedTemplates, _Receipt, _Error) ->
+    lager:debug("skipping smtp log");
+maybe_log_smtp(_, 'undefined', _Emails, _Subject, _RenderedTemplates, _Receipt, _Error) ->
+    lager:debug("skipping smtp log since account_id is 'undefined'");
+maybe_log_smtp(_, AccountId, Emails, Subject, RenderedTemplates, Receipt, Error) ->
```

Comment:
May as well match 'false' here and in the previous clause, since we know it must be that.